modApiServer/src/org/aion/api/server/rpc3/PersonalRPC.java

```
@@ -24,20 +24,20 @@ default String execute(Request request){
                 try{
                     params=EcRecoverParamsConverter.decode(request.params);
                 }catch(Exception e){
-                    throw new InvalidParamsRPCException();
+                    throw InvalidParamsRPCException.INSTANCE;
```

We should throw the exception contains.